devtools/parse_pcap_klap.py

```diff
@@ -69,18 +73,38 @@ def update_encryption_session(self):
             self._local_auth_hash = self._klap.handshake1_seed_auth_hash(
                 self._local_seed, self._remote_seed, self._auth_hash
             )
-            if (self._remote_auth_hash is not None) and (
-                self._local_auth_hash != self._remote_auth_hash
-            ):
-                raise ValueError(
-                    "Local and remote auth hashes do not match.\
-This could mean an incorrect username and/or password."
+            auth_hash = None
+            if self._remote_auth_hash is not None:
+                if self._local_auth_hash == self._remote_auth_hash:
+                    auth_hash = self._local_auth_hash
+                else:
+                    # Check whether default credentials were used.
+                    for value in DEFAULT_CREDENTIALS.values():
+                        default_credentials = get_default_credentials(value)
+                        default_auth_hash = self._klap.generate_auth_hash(
+                            default_credentials
```

Comment:
Probably a good idea to have a separate method for this check to try to keep it a bit simpler?

Reply:
That's done